hooks/useLogin.ts

```diff
@@ -116,6 +131,20 @@ export function useLogin(config: LoginProps = {}) {
   return loginContext
 }
 
+function getCookie(cname: string) {
+  const name = cname + '='
+  const ca = document.cookie.split(';')
+  for (let i = 0; i < ca.length; i++) {
+    const c = ca[i].trim()
+
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length)
+    }
+  }
+
+  return ''
+}
```

Comment:
If you want to avoid loops and go with something a little more declarative:

```ts
function getCookie(name: string) {
  return document.cookie
    .split(/;\s?/)
    .find((item) => item.startsWith(`${name}=`))
    ?.slice(name.length + 1) ?? '';
}
```